Assets/Scripts/Enemies/SpaceFly_Adult.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SpaceFly_Adult : EnemyHealth
+{
+    [SerializeField]
+    private GameObject bullet;
+
+    [SerializeField]
+    private int numBullets = 3;
+    [SerializeField]
+    private float shootTimerMin = 6.0f;
+    [SerializeField]
+    private float shootTimerMax = 14.0f;
+    private float shootTimer;
+    [SerializeField]
+    private float rateOfFire = 10f;
+
+    Vector3 playerPos;
+    private WaitForSeconds rof;
```

Comment:
I understood that this was rate of fire but i would recommend either changing the name of this variable, or of the actual rateOfFire one to something else like time to shoot